cmd/pfcpsim-client/main.go

```diff
@@ -390,15 +400,21 @@ func createSessions(count int) {
 				Build(),
 		}
 
-		// TODO keep track of new session
-		err := globalPFCPSimClient.EstablishSession(pdrs, fars, qers)
+		sess, err := globalPFCPSimClient.EstablishSession(pdrs, fars, qers)
 		if err != nil {
 			log.Errorf("Error while establishing sessions: %v", err)
 			return
 		}
 
-		// TODO show session's F-SEID
-		log.Infof("Created session")
+		activeSessions = append(activeSessions, &PFCPClientContext{
+				session: sess,
+				pdrs:    pdrs,
+				fars:    fars,
+				qers:    qers,
+			},
+		)
+
+		log.Infof("Created new PFCP session")
```

Comment:
having private fields, I cannot show session's SEID. is it ok?

Reply:
it's ok

Reply:
at least for now